core/src/main/resources/hudson/model/UsageStatistics/global.groovy

```diff
@@ -8,7 +8,7 @@ def f=namespace(lib.FormTagLib)
 
 f.section(title: _("Usage Statistics")) {
     if (UsageStatistics.DISABLED) {
-        span(class: "jenkins-not-applicable") {
+        div(class: "jenkins-not-applicable jenkins-!-margin-bottom-2") {
```

Comment:
`jenkins-description` will give the appropriate colour/spacing:

<img width="259" alt="image" src="https://github.com/user-attachments/assets/2d97c430-751e-4ebe-8453-ed22f77d6825">

Can leave `jenkins-not-applicable` if you need the italic text, don't think its adding anything though.

Reply:
```suggestion
        div(class: "jenkins-not-applicable jenkins-description") {
```